spec/features/traject/ead2_indexing_spec.rb

```diff
@@ -58,5 +58,56 @@
         )
       end
     end
+
+    it "gets the title tesim" do
+      expect(result["title_teim"]).to include(
+        "Harold B. Hoskins Papers"
+      )
+      expect(result["title_teim"]).to eq(
+        result["title_ssm"]
+      )
+    end
+
+    it "asserts that title filing si field is missing" do
+      expect(result["title_filing_si"]).to be_nil
+    end
+
+    context "YearRange normalizer tests" do
+      let(:years) { result["normalized_date_ssm"][0].split("-") }
+      let(:beginning) { years[0].to_i }
+      let(:ending) { years[1].to_i }
+
+      it "asserts YearRange normalizer works, that normalized_date_ssm contains start and end in date_range_sim field" do
```

Comment:
Then the test name should be a continuation as if it's the rest of the sentence, like "normalizes the year range into start and end dates" -- the details of the precise field names, etc, are clear by looking at the body of the test.